packages/utils/src/coverage-assessment.ts

```diff
@@ -93,29 +99,31 @@ function initCsv(fileName: string) {
   fs.writeFileSync(fileName, csvHeader);
 }
 
-async function getPatientIds(log: typeof console.log): Promise<string[]> {
+async function getPatientIds(): Promise<{ patientIds: string[]; isAllPatients: boolean }> {
   if (patientIds.length > 0) {
-    return patientIds;
+    return { patientIds, isAllPatients: false };
   }
-  return await getAllPatientIds(log);
+  const allPatientIds = await getAllPatientIds();
+  return { patientIds: allPatientIds, isAllPatients: true };
 }
 
-async function getAllPatientIds(log: typeof console.log): Promise<string[]> {
-  displayNoDryRunWarning(log);
-  log(
-    "You are about to trigger a document query for all patients of the CX. This is very expensive!"
-  );
-  log("Cancel this script now if you're not sure.");
-  await sleep(confirmationTime.asMilliseconds());
+async function getAllPatientIds(): Promise<string[]> {
   const resp = await axios.get(`${apiUrl}/internal/patient/ids?cxId=${cxId}`);
   const patientIds = resp.data.patientIds;
   return (Array.isArray(patientIds) ? patientIds : []) as string[];
 }
 
-function displayNoDryRunWarning(log: typeof console.log) {
-  // The first chars there are to set color red on the terminal
-  // See: // https://stackoverflow.com/a/41407246/2099911
-  log("\n\x1b[31m%s\x1b[0m\n", "---- ATTENTION - THIS IS NOT A SIMULATED RUN ----");
+async function displayWarningAndConfirmation(
```

Comment:
Could this be imported? Would just need to hardcode isDryRun to false within this script.

Reply:
Each "display warning" function has a specific logic, but I moved the "attention" log w/ colouring to an external function and made the scripts point to it.